sormas-cargoserver/redeployUI.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+here=$(dirname $0)
+
+pushd $here/../sormas-ui/
+mvn clean install -DskipTests
+popd
+
+sormasUi=`sh $here/target/cargo/installs/payara-5.194/payara5/glassfish/bin/asadmin list-applications --user admin --port 6048 --passwordfile $here/password.txt | grep sormas-ui | sed "s/^\([^ ]*\) .*$/\1/"`
+echo "--- redeploy $sormasUi ---"
+sh $here/target/cargo/installs/payara-5.194/payara5/glassfish/bin/asadmin --user admin --port 6048 --passwordfile $here/password.txt undeploy $sormasUi
```

Comment:
@syntakker Typically we (re)deploy using the autodeploy folder of the domain, so you don't need to make use of asadmin. I guess this is not possible here, because the cargo server uses a different approach for the initial deployment, right?

Please extend the readme to reflect on the existence of this script and how/when to use it.

Reply:
@MartinWahnschaffeSymeda you're right, apart from being for a very special purpose, the script contradicts the autodeploy strategy. I removed the script.